src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -149,9 +161,17 @@
 
 (schema/defn ^:always-validate get-jvm-metrics :- JvmMetricsV1
   []
-  (let [runtime-bean (ManagementFactory/getRuntimeMXBean)]
+  (let [runtime-bean (ManagementFactory/getRuntimeMXBean)
+        gc-beans (jmx/mbean-names "java.lang:name=*,type=GarbageCollector")]
     {:heap-memory (jmx/read "java.lang:type=Memory" :HeapMemoryUsage)
      :non-heap-memory (jmx/read "java.lang:type=Memory" :NonHeapMemoryUsage)
+     :file-descriptors (clojure.set/rename-keys
```

Comment:
I thought about that, but `:as set` seems like it might shadow `clojure.core/set`. Couldn't think of a better alias, so I just went with the namespace since it's pretty short to begin with.